xscen/regridding.py

```diff
@@ -25,6 +27,7 @@ def regrid(
     ds_grid: xr.Dataset,
     *,
     regridder_kwargs: Optional[dict] = None,
+    intermediate_reg_grids: Optional[dict] = None,
```

Comment:
I think `intermediate_grids` is simpler and would work?

Reply:
name changed!